kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CrudInformerTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2020 VMware, Inc.
```

Comment:
I think license check is failing due to this :-(

Reply:
Right now we have a fixed license header which includes Red Hat as the copyright owner. We faced similar issues when accepting contributions from an Oracle employee in #2030

Reply:
This is an unusual requirement. What's the fix?

Reply:
For now, Would it be possible for you to change owner to Red Hat? We do have plans to make license headers generic( #2045)

Reply:
Thanks. Would it be possible to just add this test case to an existing test class instead? Keeps things simple. :)

Reply:
We can do that, but I don't remember if we have any test class for testing informers in crud mode :roll_eyes: . Do you think it's a good idea moving it here: https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/DefaultSharedIndexInformerTest.java ?

Reply:
Fixed.